SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py

```diff
@@ -57,48 +57,29 @@
       EfficiencyFactors_Barrel = cms.vdouble(0.999, 0.999, 0.999, 0.999, 0.999, 0.999, 0.999, 0.999, 0.999, 0.999 ),
       EfficiencyFactors_Endcap = cms.vdouble(0.999, 0.999, 0.999, 0.999, 0.999, 0.999, 0.999, 0.999, 0.999, 0.999, 0.999, 0.999, 0.999, 0.999, 
       0.999, 0.999 ),#Efficiencies kept as Side2Disk1,Side1Disk1 and so on
-      CellsToKill = cms.VPSet()
-    ),
-#Pixel-3D Digitizer Algorithm
-    Pixel3DDigitizerAlgorithm = cms.PSet(
```

Comment:
@suchandradutta why are you removing the configuration for the 3D pixel digitizer algorithm?
@duartej

Reply:
Thanks @mmusich for noticing this!! 
I saw more removals related with the pixel 3D  in this PR, could you please revert this changes @suchandradutta ?